src/System.Private.Xml/src/System/Xml/DiagnosticsSwitches.cs

```diff
@@ -7,7 +7,8 @@ namespace System.Xml
     using System.Diagnostics;
 
 #if DEBUG
-    public static class DiagnosticsSwitches {
+    public static class DiagnosticsSwitches
```

Comment:
Does this ever actually need to be public, or could this debug-specific code be removed?